api_aspnet-core_csharp_hello-world/HelloworldApplication/appsettings.json

```diff
@@ -9,7 +9,7 @@
   "AllowedHosts": "*",
   "Urls": "http://localhost:6060;https://localhost:6061",
   "Auth0": {
-    "Domain": "{DOMAIN}",
+    "Domain": "dev-zeekjasj.us.auth0.com",

```

Comment:
Did you mean to replace the placeholder here?

Reply:
Ops! Good catch, @mraible. Thank you